hbase/hbase/stackable/patches/2.6.1/0006-Update-property-usage-for-bound-ports.patch

```diff
@@ -0,0 +1,174 @@
+index ea96ff8fce..054beb10d3 100644
+--- a/hbase-common/src/main/java/org/apache/hadoop/hbase/HConstants.java
++++ b/hbase-common/src/main/java/org/apache/hadoop/hbase/HConstants.java
+@@ -212,7 +212,7 @@ public final class HConstants {
+   /** Configuration key for advertised master web API port */
+   public static final String MASTER_INFO_PORT = "hbase.master.info.port";
+ 
+-  /** Configuration key for bound master web API port. (Defaults to MASTER_INFO_PORT.) */
++  /** Configuration key for bound master web API port */
+   public static final String MASTER_BOUND_INFO_PORT = "hbase.master.bound.info.port";
+ 
+   /** Configuration key for the list of master host:ports **/
+@@ -340,7 +340,7 @@ public final class HConstants {
+   /** Configuration key for advertised region server web API port */
+   public static final String REGIONSERVER_INFO_PORT = "hbase.regionserver.info.port";
+ 
+-  /** Configuration key for bound region server web API port. (Defaults to REGIONSERVER_INFO_PORT.) */
++  /** Configuration key for bound region server web API port */
+   public static final String REGIONSERVER_BOUND_INFO_PORT = "hbase.regionserver.bound.info.port";
+ 
+   /** A flag that enables automatic selection of regionserver info port */
+diff --git a/hbase-server/src/main/java/org/apache/hadoop/hbase/LocalHBaseCluster.java b/hbase-server/src/main/java/org/apache/hadoop/hbase/LocalHBaseCluster.java
+index 816ef997cb..2114725986 100644
+--- a/hbase-server/src/main/java/org/apache/hadoop/hbase/LocalHBaseCluster.java
++++ b/hbase-server/src/main/java/org/apache/hadoop/hbase/LocalHBaseCluster.java
+@@ -144,20 +144,20 @@ public class LocalHBaseCluster {
+       // treat info ports special; expressly don't change '-1' (keep off)
+       // in case we make that the default behavior.
+       if (
+-        conf.getInt(HConstants.REGIONSERVER_INFO_PORT, 0) != -1
+-          && conf.getInt(HConstants.REGIONSERVER_INFO_PORT,
++        conf.getInt(HConstants.REGIONSERVER_BOUND_INFO_PORT, 0) != -1
++          && conf.getInt(HConstants.REGIONSERVER_BOUND_INFO_PORT,
+             HConstants.DEFAULT_REGIONSERVER_INFOPORT) == HConstants.DEFAULT_REGIONSERVER_INFOPORT
+       ) {
+         LOG.debug("Setting RS InfoServer Port to random.");
+-        conf.set(HConstants.REGIONSERVER_INFO_PORT, "0");
++        conf.set(HConstants.REGIONSERVER_BOUND_INFO_PORT, "0");
+       }
+       if (
+-        conf.getInt(HConstants.MASTER_INFO_PORT, 0) != -1
+-          && conf.getInt(HConstants.MASTER_INFO_PORT, HConstants.DEFAULT_MASTER_INFOPORT)
++        conf.getInt(HConstants.MASTER_BOUND_INFO_PORT, 0) != -1
++          && conf.getInt(HConstants.MASTER_BOUND_INFO_PORT, HConstants.DEFAULT_MASTER_INFOPORT)
+               == HConstants.DEFAULT_MASTER_INFOPORT
+       ) {
+         LOG.debug("Setting Master InfoServer Port to random.");
+-        conf.set(HConstants.MASTER_INFO_PORT, "0");
++        conf.set(HConstants.MASTER_BOUND_INFO_PORT, "0");
```

Comment:
(And here)

Reply:
That was changed inadvertently - have reverted it: 65aaae2